resources/terraform/aws-eks/outputs.tf

```
@@ -0,0 +1,10 @@
+output "kubeconfig_path" {
+  description = "Kubeconfig path for the eks cluster"
+  sensitive   = true
+  value       = var.kubeconfig_path
+}
+
+output "repo_url" {
```

```suggestion
output "repository" {
```

It might make sense for all of these repository output names to match (see https://github.com/dominodatalab/testenv/blob/3275dc58c941410a021285b0acba681cd8350ef4/resources/terraform/gcp-gke/outputs.tf#L7) for consistency's sake. It's not that we can't map in the Go types, but it just seems like since we control the output name, then why not make them uniform, right?